resources/js/components/ui/Table/TQueryTable.tsx

```diff
@@ -190,20 +192,25 @@ export const TQueryTable: VoidComponent<TQueryTableProps> = (props) => {
     const ignoreColumnsSet = new Set(props.ignoreColumns || []);
     const schemaColumns = sch.columns.filter(({name}) => !ignoreColumnsSet.has(name));
     const columns = [
-      ...schemaColumns.map(({type, nullable, name}) => {
-        const common = commonColumnDef(name, type);
-        return h.accessor(name, {
-          ...common,
-          meta: {
-            ...common.meta,
-            tquery: {
-              type,
-              nullable,
-              ...columnOptions(name).metaParams,
-            } satisfies TQueryColumnMeta,
-          },
-        });
-      }),
+      ...schemaColumns
+        .map(({type, nullable, name}) => {
+          if (!isDataType(type)) {
+            return undefined;
+          }
+          const common = commonColumnDef(name, type);
+          return h.accessor(name, {
+            ...common,
+            meta: {
+              ...common.meta,
+              tquery: {
+                type,
+                nullable,
+                ...columnOptions(name).metaParams,
+              } satisfies TQueryColumnMeta,
+            },
+          });
+        })
+        .filter(NON_NULLABLE),
```

Comment:
tylko not null kolumny?

Reply:
Nie no, to łapie ten undefined dla kolumny count. To jest taki `.filter(e=>!!e)` tylko lepiej otypowany.